pages/analysis-sets/[id].js

```diff
@@ -82,6 +83,31 @@ export default function AnalysisSet({
               )}
               <DataItemLabel>File Set Type</DataItemLabel>
               <DataItemValue>{analysisSet.file_set_type}</DataItemValue>
+              {analysisSet.summary && (
+                <>
+                  <DataItemLabel>Summary</DataItemLabel>
+                  <DataItemValue>{analysisSet.summary}</DataItemValue>
+                </>
+              )}
+              {analysisSet.functional_assay_mechanisms?.length > 0 && (
```

Comment:
I don’t _think_ you need the question mark. It looks like `functional_assay_mechanisms` always exists, though it could be empty.

Reply:
removed.